### ISSUE TITLE
CRM 2016 Support

### ISSUE THREAD

**Issue by DigitalFlow** [jordimontana82/fake-xrm-easy#17]
Hey Jordi,

since CRM 2016 was released we should also publish FakeXRMEasy packages supporting this version.
Do you have time for this or should I do it and send a PR?

Kind Regards

**Comment by jordimontana82** [comment]
:+1:  Totally agree with that.

With that version it was also introduced a new web api which uses REST which we need to review and add it to the package too.

Will be busy with the other PBIs in the backlog so feel free to add a new PR for that.

Please, keep in mind when adding it:

1) Projects for the different versions use links to existing files in the 2011 project (like FakeXrmEasy.2013 and FakeXrmEasy.2015). This is because there are subtle differences across versions and wanted to reuse as much code as possible across the 2011, 2013 and 2015 versions. That's why you'll see links to files and also conditional compilation for the small different parts. For example, there are symbols for 2013 FAKE_XRM_EASY_2013 and similar. These are defined in the project settings.

2) Same concept applies to the test projects. Link to files and precompilation symbols.

3) Finally, I was using the same assembly version for all the different packages, again, because a fix might apply to the latest version and older ones and it might get really confusing and difficult to mantain different versions in parallel. Just to keep it simple.

**Comment by jordimontana82** [comment]
Hey @DigitalFlow ,
 
Will add this myself. We need also an implementation for the Associate and Disassociate messages, which are pretty basic.I raised an issue for it. Do you want to have a look at that? :)
